manifests/4.8/sriov-network-operator.v4.8.0.clusterserviceversion.yaml

```diff
@@ -371,6 +371,8 @@ spec:
   labels:
     olm-owner-enterprise-app: sriov-network-operator
     olm-status-descriptors: sriov-network-operator.v4.8.0
+    operatorframework.io/arch.amd64: supported
```

Comment:
There are several operands managed by SR-IOV Operator, do you think we need to label them with the same tags?(I'm not very famiilar with how multi-arch works, any reference would be helpful here).

Reply:
> There are several operands managed by SR-IOV Operator, do you think we need to label them with the same tags?(I'm not very famiilar with how multi-arch works, any reference would be helpful here).

It is only required for the operator, and not all the operands.